.pubnub.yml

```diff
@@ -0,0 +1,168 @@
+---
+name: eon-map
+scm: https://github.com/pubnub/eon-map
+version: "1.2.2"
+schema: 1
+files:
+  - ./eon-map.js
+changelog:
+  - version: 1.2.2
+    date: 2017-07-08
+    changes:
+      - type: improvement
+        text: "Remove bad error checks."
+  - version: 1.2.1
+    date: 2017-07-08
+    changes:
+      - type: bug
+        text: "Fix require path for `mapbox.js`."
+      - type: improvement
+        text: "Add channels / groups verifications."
+      - type: improvement
+        text: "Add default map zoom and center location."
+  - version: 1.2.0
+    date: 2017-07-07
+    changes:
+      - type: improvement
+        text: "Move from Bower builds to Webpack"
+      - type: improvement
+        text: "Update keys used by demos."
+  - version: 1.1.4
+    date: 2017-06-02
+    changes:
+      - type: bug
+        text: "Fix mapbox dependency."
+  - version: 1.1.3
+    date: 2017-06-02
+    changes:
+      - type: improvement
+        text: "Update mapbox to v3"
+  - version: 1.1.1
+    date: 2017-06-02
+    changes:
+      - type: feature
+        text: "Add Leaflet Mutant plugin to display baselayers using mutation observers."
+  - version: 1.1.0
+    date: 2017-06-02
+    changes:
+      - type: feature
+        text: "Add Google maps support."
+features:
+  channel-groups:
+    - CHANNEL-GROUPS-LIST-CHANNELS-IN-GROUP
+  storage:
+    - STORAGE-INCLUDE-TIMETOKEN
+    - STORAGE-COUNT
+    - STORAGE-FETCH-MESSAGES
+  subscribe:
+    - SUBSCRIBE-CHANNELS
+    - SUBSCRIBE-CHANNEL-GROUPS
+    - SUBSCRIBE-PUBSUB-V2
+sdks:
+  - full-name: EON Maps Framework
+    short-name: EON Maps
+    artifacts:
+      - artifact-type: framework
+        language: JavaScript
+        tier: 3
+        tags: 
+          - Web
+        source-repository: https://github.com/pubnub/eon-map
+        documentation: https://www.pubnub.com/developers/eon
+        distributions:
+          - distribution-type: source
```

Comment:
this should be api-client

Reply:
Distribution type? Usually it is package, source

Reply:
Source is not there in the enum (and source is equivalent to `api-client`)

                        "api-client",                         "library",                         "package",                        "application",                        "framework"

Reply:
Where?

https://github.com/pubnub/sdk-specifications/blob/master/metadata/schema/pubnub-yml-validator.schema.json#L397-L400

Reply:
Please check artifact-type, I will double check if we need the same enum here as well

Reply:
But you left comment on distribution type (or GitHub app for phone is buggy)

Reply:
ya I mentioned it on `distribution type`, and it should use the same enums from `artifact-type`.  (I will double check though)